LaphamsQuarterly.org.js

```diff
@@ -0,0 +1,528 @@
+{
+	"translatorID": "e329ec79-397e-4aa5-a06e-1aa32f10a138",
+	"label": "LaphamsQuarterly.org",
```

Comment:
This can just be `Lapham’s Quarterly` (typographic quote), and the filename can be `Lapham's Quarterly.js` (straight quote).

Reply:
Ah, sorry, it's the label that determines the filename, and if it's not ASCII it will use the translatorID. So can use `Lapham's Quarterly` for the label and the filename should be set to `Lapham's Quarterly.js` automatically. (We should change it to automatically convert curly quotes to straight quotes when generating the filename.)

Reply:
https://github.com/zotero/zotero/blob/e024449732f0aec01a2e92f1c71f1fe20e42607b/chrome/content/zotero/xpcom/translation/translators.js#L488-L500

Called from `Zotero.Translators.Translators.save()`: https://github.com/zotero/zotero/blob/e024449732f0aec01a2e92f1c71f1fe20e42607b/chrome/content/zotero/xpcom/translation/translators.js#L558-L560